protocol.md

```diff
@@ -334,6 +335,34 @@ Since metadata can contain arbitrary binary values, Servers SHOULD
 carefully validate metadata values or sanitize them before using them
 as header values to avoid header smuggling.
 
+Clients and servers SHOULD implement the metadata key ``filetype``
+with the value containing the actual ``Content-Type`` of the Upload
+(because the ``Content-Type`` header is formally required to be set to
+``application/offset+octet-stream`` for [Creation with
+Upload](#creation-with-upload)).
+
+Clients and servers MAY support additional well known metadata keys:
+
+* ``filename`` for a common file name
+
+The specific metadata keys documented herein are reserved for the
+respective use and MUST NOT be used for other purposes.
+
+##### [Content-Encoding](https://httpwg.org/specs/rfc7231.html#header.content-encoding)
+
+Clients MUST set the ``Content-Encoding`` header correctly IFF a
+content encoding is used.
+
+As per RFC7231, Servers MUST respond with status 415 if they can not
+accept the ``Content-Encoding`` chosen by the client.
+
+Servers MUST either store the ``Content-Encoding`` and deliver it with
+subsequent requests, or properly decode the content before storing it.
```

Comment:
> deliver it with subsequent requests

Does this mean that the Content-Encoding header must be present in the response for every PATCH, DELETE and HEAD request?